### ISSUE TITLE
Configuration option for single modal button

### ISSUE THREAD

**Issue by pbelch** [predixdesignsystem/px-modal#39]
Hi.

I am trying to use a px-modal that has only a single 'Cancel' button rather than an accept and reject button.

Is it possible to do this? (I cannot see an option in the docs)

Currently i have implemented the following custom content with a single button, but still see an empty button when i load the page:

` <px-modal
        id= "machineCreatedDetails"
        modal-heading="Machine Created">
        <div class = "modal" slot="header">
           <h2>Machine Created</h2></div>
        <div class = "modal" slot="body">
            Your machine has been created. In order to send data to the application, you will require the followig information:
            <p>
              <b>Machine ID: </b>{{newMachineID}}<br/>
              <b>API Key: </b>{{newAPIKey}}
            </p>
            <p>
                PLEASE MAKE A NOTE OF THIS INFORMATION AS YOU WILL NOT SEE THIS INFORMATION AGAIN
            </p>
        </div> 
        <button class="btn btn--call-to-action" slot="reject-trigger">Close</button>
      </px-modal> `

which gives me output that looks like this:
<img width="582" alt="modal-two-buttons" src="https://user-images.githubusercontent.com/9989676/38028787-dae94a4e-328b-11e8-8a01-aac3cf3f8992.PNG">

I am wanting to disable the second button if there is a configuration option for this?

**Comment by sgoodrow** [comment]
We have the same requirement, except we'd like to extend it to:
1) Allow hiding both buttons (application is busy, modal will close automatically) or allow disabling of both buttons (application is busy, control will be returned soon).

**Comment by sgoodrow** [comment]
Ok, I believe this can be solved as-is:
**You can override the button definitions and hide them or disable them.**
```
<px-modal ...>
...
      <button class="hidden" slot="reject-trigger" :disabled.prop='true' id="rejectButton">REJECT BUTTON TEXT</button>
      <button class="btn" slot="accept-trigger" :disabled.prop='true' id="acceptButton">ACCEPT BUTTON TEXT</button><px-modal/>

.hidden {
      display: none;
}
```

<img width="465" alt="modalwithbuttonshiddenordisabled" src="https://user-images.githubusercontent.com/5815812/38905538-1fdf98b2-4266-11e8-84f2-f4f01e5e51f9.png">

**Comment by pbelch** [comment]
Yea i've done something similar (not writing a vue app), and have included the button, setting `display:none` in the css

**Comment by arongraham-ge** [comment]
@pbelch we have released v3.3.5 that adds two new `hide-accept-button` and `hide-reject-button`. We have updated the docs and demo on https://www.predix-ui.com/#/elements/px-modal so please update to the new release and let us know if this works for your use case or you have any issues.
